src/main/java/com/linagora/james/mailets/json/ClassificationRequestBody.java

```diff
@@ -93,13 +96,15 @@ private static int getSanitizedMailSize(Mail mail) throws MessagingException {
     private final Recipients recipients;
     private final List<String> subject;
     private final String textBody;
+    private final Optional<ZonedDateTime> date;
 
-    private ClassificationRequestBody(UUID messageId, List<Emailer> from, Recipients recipients, List<String> subject, String textBody) {
+    private ClassificationRequestBody(UUID messageId, List<Emailer> from, Recipients recipients, List<String> subject, String textBody, Optional<ZonedDateTime> date) {
```

Comment:
That depends if you call it only once or twice then yes doing a builder would be useless